githubteacher/USERNAME-githubteacher-kansascity.md

```diff
@@ -0,0 +1,5 @@
+# Kansas City
+
+### Restaurants
```

Comment:
@githubteacher Any places that take pets? My dog's hungry after skydiving yesterday
![2572294_f520](https://cloud.githubusercontent.com/assets/3360602/16277813/795e7e66-3882-11e6-80a2-37591a6fec1f.jpg)